lib/AppInfo/Capabilities.php

```diff
@@ -21,6 +23,10 @@ public function __construct(
 	) {
 	}
 
+	/**
+	 * @throws Exception
+	 * @throws RequestBuilderException
+	 */
```

Comment:
Should be caught inside, this method is not documented as throwing in ICapability and should not throw.